resources/migrations/V1.0.4__add_cleanup_trashcan_for_backup.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE backups
+    ADD trashcan_cleanup_status TEXT DEFAULT 'Noop';
+ALTER TABLE backups
+    ADD trashcan_cleanup_error_message TEXT DEFAULT NULL;
+ALTER TABLE backups
+    ADD trashcan_cleanup_last_scheduled_timestamp TEXT DEFAULT NULL;
```

Comment:
Why is the timestamp a TEXT type and not timestamp or datetime?

Reply:
noooo copy paste